modules/order/commerce_order.module

```diff
@@ -157,6 +157,49 @@ function commerce_order_add_line_items_field($order_type) {
 }
 
 /**
+ * Adds the default adjustments field to an order type.
+ *
+ * Adjustments can't be a base field because the Views integration is broken.
+ * Instead, it is created as a configurable field for each order type.
+ *
+ * @param \Drupal\commerce_order\Entity\OrderTypeInterface $order_type
+ *   The order type.
+ */
+function commerce_order_add_adjustments_field($order_type) {
```

Comment:
No, adjustments can be a base field just fine :) We're not going to add views relationships based on any adjustment data, it's all serialized.